lesson08/index.js

```diff
@@ -17,17 +17,23 @@ function createListDom(itemList) {
     ulElement.appendChild(frag);
 }
 
+function createLoading() {
+    const bodyElement = document.querySelector('.body');
+    bodyElement.insertAdjacentHTML('afterbegin', '<div class="loading"><img class="loading-img" src="./img/loading-circle.gif"></div></div>');
+}
+
 function removeLoading() {
-    const loadingImage = document.querySelector(".js-loading");
-    loadingImage.classList.add("loaded");
-    // loadingImage.parentNode.removeChild(loadingImage);
+    const loadingImage = document.querySelector(".loading");
+    // loadingImage.classList.add("loaded");
+    loadingImage.parentNode.removeChild(loadingImage);
 }
```

Comment:
Q
> 'removeLoading' is defined but never used. (no-unused-vars)

Warning is issued on CodeSandbox, but is this intentionally left?

Reply:
I forgot to comment out.
Since it is a function that will be used after this, I commented it out once.

Latest commit:ac69d94

https://codesandbox.io/s/modest-khorana-yicng?file=/index.js

Reply:
Deleted all unnecessary descriptions

Latest commit:
ac69d94

codesandbox
https://codesandbox.io/s/modest-khorana-yicng?file=/index.js